src/Chapter21/Listing21.01.SynchronousPiSections.cs

```diff
@@ -21,6 +22,25 @@ public static void Main()
             Console.WriteLine(pi);
         }
     }
+    #region EXCLUDE
+    /*
+    #endregion EXCLUDE
+    using System;
+
+    public static class PiCalculator
+    {
+        public static string Calculate(
+            int digits, int startingAt)
+        {
+            //...
+        }
+
+        //...
+    }
+#region EXCLUDE
+    */
```

Comment:
```suggestion
    }
    #region EXCLUDE
    */
```
nit: make this on the same indentation as everything else